### DISCUSSION TITLE
Creality 4.2.7 trouble with old ender 3 display

### DISCUSSION THREAD

**Discussion by fluxcore488** [Jyers/Marlin#1181] [Q&A] [answered]
I am having quite a bit of trouble flashing the default bin..It just hangs and the display shows nothing. then i have to reflash with the creality bin to recover. I have tryed 3 times now..anyone have any ideas?

**Comment by Cal-LF** [comment, marked as answer]
Are you trying to install Jyres firmware using an ender 3 display?

**Reply by fluxcore488** [reply]
yup...I dont want the new one.

**Reply by Cal-LF** [reply]
I'm pretty sure Jyres firmware only supports the E3V2 display.

**Reply by fluxcore488** [reply]
Well that explanes that i guess. Sad because i want to run arkwelder

**Reply by Cal-LF** [reply]
What does the Ender 3 display have to do with using Arc Welder?

**Reply by fluxcore488** [reply]
I need a diffrent firmware on the 4.2.7 to run it.I cant seem to get the E3VT default to load and i was wondering if it was because i have the old display .
